js/ng.js

```diff
@@ -463,4 +544,92 @@ async function deleteGloss(id=glossHashID) {
         console.log(err)
     })
 
-}
\ No newline at end of file
+}
+
+async function queryBibliographicCitations(glossId) {
+    const query = {
+        "@type": "BibliographicCitation",
+        "references": glossId, 
```

Comment:
Careful here.  We use `references` elsewhere but track it as an Array, even when only a single ID.